index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="style.css">
```

Comment:
meta tags are typically used to specify a character set, page description, keywords, author of the document, and viewport settings. therefore it would be better to place the style links below the meta tag so that your code looks professional.